ntc_templates/templates/juniper_junos_show_system_processes_brief.textfsm

```diff
@@ -0,0 +1,23 @@
+Value NODE (\D+\d+)
```

Comment:
We can simplify this node regex.
_(There should be a button to "commit suggestion" to accept my suggestion, which should make that process simple.)_

```suggestion
Value NODE (\S+)
```